common/src/test/resources/application.common.properties

```diff
@@ -9,6 +9,7 @@ spring.datasource.password=${DB_PASSWORD}
 spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.show-sql=true
```

Comment:
In my experience, generally this is not done in production all the time.  This is a lower environment setting or for a critical bug.